src/Entity/NphDlw.php

```diff
@@ -48,6 +48,9 @@ class NphDlw
     #[ORM\Column(length: 50, nullable: true)]
     private ?string $visitPeriod = null;
 
+    #[ORM\Column(type: Types::TEXT, nullable: true)]
+    private ?string $rdr_id = null;
```

Comment:
Could you use camel case (rdrId) instead of an underscore? You might need to update all the getters, setters, and other related places as well.

Reply:
The database field can remain rdr_id, but in the entity, you can name it rdrId.